ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/RecordNode.java

```diff
@@ -61,10 +63,20 @@ public String topicName() {
     return topicName;
   }
 
+  @SuppressWarnings("unchecked")
   public Record build(final Map<String, Topic> topics) {
     final Topic topic = topics.get(topicName());
 
-    final Object topicValue = buildValue(topic);
+    Object topicValue = buildValue(topic);
+
+    if (topicValue instanceof Map) {
+      final Map<String, Object> map = (Map<String, Object>) topicValue;
+      final Map<String, Object> uppercaseMap = new LinkedHashMap<>();
```

Comment:
why are we using a `LinkedHashMap` here?

Reply:
It should be just a simple HashMap! Changed it.